README.md

```diff
@@ -100,10 +99,4 @@ See the [Contributing](CONTRIBUTING.md) guide for details.
 
 This project is licensed under the [MIT License](LICENSE.md).
 
----
 
-Cross-browser testing provided by:
-
-<a href="https://www.browserstack.com">
-  <img height="70" src="docs/browserstack-logo.svg" alt="BrowserStack">
-</a>
```

Comment:
Couldnt get browserstack working. Unsure if they're fully paid-only at this point

Reply:
As IE11 is no longer supported in the versions of Ember this addon supports, browserstack was primarily being used for that testing, therefor :poof: its gone